packages/js/e2e-utils-playwright/src/editor.js

```diff
@@ -0,0 +1,164 @@
+const { expect } = require( '@playwright/test' );
+
+export const closeChoosePatternModal = async ( { page } ) => {
+	const closeModal = page
+		.getByLabel( 'Scrollable section' )
+		.filter()
+		.getByRole( 'button', {
+			name: 'Close',
+			exact: true,
+		} );
+	await page.addLocatorHandler( closeModal, async () => {
```

Comment:
I don't think this will work as it's a custom method and isn't defined in the package.

Reply:
@Sidsector9 Did you confirm that this works?  `addLocatorHandler` is a custom method, and I didn't see it defined anywhere in the package.

Reply:
@lanej0 `addLocatorHandler` is a builtin Playwright method. We're good. https://playwright.dev/docs/api/class-page#page-add-locator-handler